src/components/dashboard/FeedModalList.js

```diff
@@ -143,6 +145,9 @@ const getStylesFromProps = ({ theme }) => ({
     maxWidth: '100vw',
     flex: 1,
   },
+  horizontalListItem: {
+    width: '100vw',
```

Comment:
1. see if we can use 100%
2. why do we suddenly need this? what changed?

Reply:
I tried to use 100% and it gives no effect. So I used 100vw to stretch the feed modal item to whole screen width. I made it to fix the positioning of the popup as before I could see two or more items at the same time, the width of the separate feed item was not correct.

Reply:
1. we need another solution not based on vw
2. i still dont understand what changed, the modals were perfectly fine, why are they suddenly not displayed correctly?

Reply:
@sirpy I pushed commit with new changes.